### PR TITLE
[vpa] enable to use custom repository for clean up image

### PR DESCRIPTION
**Pull request by yasintahaerol**
**Why This PR?**
_a short description of why this PR is needed_
This image was pulling from a constant repository. Some developers or organizations want to use own their repository. It will be enabled to use custom repository by this feature.

Fixes #
It will be enabled to use custom repository by this feature.

**Changes**
Changes proposed in this pull request:

*
*

**Checklist:**

* [x] I have updated the chart version in `Chart.yaml` following Semantic Versioning.
* [x] Any new values are backwards compatible and/or have sensible default.
* [x] Any new values have been added to the README for the Chart, or helm-docs has been run for the charts that support it.

### CODE REVIEW

**Comment by yasintahaerol** [comment]
Hi, is everything okay? @sudermanjr

**Review by sudermanjr — CHANGES_REQUESTED** [review]
This looks like it will not be backwards compatible with old versions of the values file.

I think we either need to use a different structure for the values, or bump the major version

**Comment by yasintahaerol** [comment]
I think we should bump the major version.

**Review by lucasreed — CHANGES_REQUESTED** [review]
@yasintahaerol Agreed, can you please bump the version to `1.0.0`. Can you add a section to the readme documenting this breaking change as well? Thanks!

**Comment by yasintahaerol** [comment]
hey @lucasreed. I think [this](https://app.circleci.com/pipelines/github/FairwindsOps/charts/3049/workflows/74985554-299e-439a-b3d2-84404080016f/jobs/16547?invite=true#step-103-30) error is not related to my commit. How should we proceed?

File: stable/vpa/Chart.yaml [hunk 1]
```diff
@@ -2,7 +2,7 @@ apiVersion: v2
 name: vpa
 description: A Helm chart for Kubernetes Vertical Pod Autoscaler
 type: application
-version: 0.5.0
-appVersion: 0.9.2
+version: 0.5.1
```
Comment (sudermanjr) [inline stable/vpa/Chart.yaml]:
```suggestion
version: 1.0.0
```

File: stable/vpa/Chart.yaml [hunk 2]
```diff
@@ -2,7 +2,7 @@ apiVersion: v2
...
+appVersion: 1.0.0
```
Comment (sudermanjr) [inline stable/vpa/Chart.yaml]:
```suggestion
appVersion: 0.9.2
```

**Review by sudermanjr — CHANGES_REQUESTED** [review]

File: stable/vpa/values.yaml [hunk 3]
```diff
@@ -116,8 +116,14 @@ admissionController:
     env: {}
     # admissionController.certGen.resources -- The resources block for the certgen pod
     resources: {}
-  # admissionController.cleanupOnDelete -- If true, a post-delete job will remove the mutatingwebhookconfiguration and the tls secret for the admission controller
-  cleanupOnDelete: true
+  # admissionController.cleanupOnDelete.enabled -- If true, a post-delete job will remove the mutatingwebhookconfiguration and the tls secret for the admission controller
+  cleanupOnDelete:
+    enabled: true
+    image:
+      # admissionController.cleanupOnDelete.repository -- The location of the post-delete image
```
Comment (sudermanjr) [inline stable/vpa/values.yaml]:
```suggestion
      # admissionController.cleanupOnDelete.image.repository -- The repository of the post-delete image
```

File: stable/vpa/values.yaml [hunk 4]
```diff
@@ -116,8 +116,14 @@ admissionController:
...
+      repository: quay.io/reactiveops/ci-images
+      # admissionController.cleanupOnDelete.tag -- Overrides the image tag whose default is the chart appVersion
```
Comment (sudermanjr) [inline stable/vpa/values.yaml]:
```suggestion
      # admissionController.cleanupOnDelete.image.tag -- The image tag to use for the admission controller cleanup image
```

**Review by sudermanjr — CHANGES_REQUESTED** [review]

**Review by sudermanjr — APPROVED** [review]

**Review by lucasreed — APPROVED** [review]

**Pull request merged**